pkgs/halo2-eth-membership-worker/src/worker.ts

```diff
@@ -19,21 +21,20 @@ export const Halo2EthMembershipWorker: IHalo2EthMembershipWorker = {
     const response = await fetch(wasmModuleUrl)
     const bufferSource = await response.arrayBuffer()
 
-    await halo2EthMembershipWasm.initSync(bufferSource)
-    await halo2EthMembershipWasm.initPanicHook()
+    halo2EthMembershipWasm.initSync(bufferSource)
+    halo2EthMembershipWasm.initPanicHook()
 
     if (!initialized) {
       const numThreads = navigator.hardwareConcurrency
       await halo2EthMembershipWasm.initThreadPool(numThreads)
 
-      // Please note that K = 14 only supported for now until benchmarking is finished
-      params = await fetchKzgParams(14)
+      params = await fetchKzgParams(K)
 
       initialized = true
     }
   },
 
-  async proveMembership({ merkleProofBytesSerialized, message, sig }): Promise<Uint8Array> {
+  async proveMembership({ merkleProofBytesSerialized, message, sig }) {
```

Comment:
mostly a style change for consistency:
The convention of this repo is to not explicitly type functions return type and let typescript figure it out itself. (debatable practice I know ^^)

Reply:
In Rust that is not the case as far as I know :sweat_smile: